src/ParallelAlgorithms/LinearSolver/AsynchronousSolvers/ElementActions.hpp

```diff
@@ -81,18 +81,15 @@ void contribute_to_residual_observation(
       std::vector<std::string>{"Iteration", "Residual"},
       reduction_data{iteration_id, residual_magnitude_square});
   if (UNLIKELY(get<logging::Tags::Verbosity<OptionsGroup>>(cache) >=
-               ::Verbosity::Verbose)) {
+               ::Verbosity::Debug)) {
     if (iteration_id == 0) {
-      Parallel::printf(
-          "Linear solver '%s' initialized on element %s. Remaining local "
-          "residual: %e\n",
-          Options::name<OptionsGroup>(), get_output(array_index),
-          sqrt(residual_magnitude_square));
+      Parallel::printf("%s %s initialized with local residual: %e\n",
```

Comment:
Do you think it would be a good idea to keep "Linear solver" in the message so people looking at the log know what part of the code the message is from?

Reply:
Hmm, you mean print them like `[ElementId] Linear solver GMRES(IterationId): ...`? I think it's better to make the name of the solver reasonably descriptive to identify where the messages are coming from. E.g. I'll name them `SchwarzSmoother` and `Multigrid`.

Reply:
I guess my real question is: Is everyone who runs the executable expected to know that "Multigrid" is a linear solver?  If yes, then this is fine.

Reply:
I think when they read "multigrid" then they can go look up the multigrid code, which happens to be a linear solver. Sound ok?

Reply:
I'm more worried about the other direction:  I want to read the log entries about the linear solver, but I don't know which ones those are.  I may be overestimating how much of a problem that will be, though, so let's wait and see how things look when we have everything fully running.